README.md

```diff
@@ -731,7 +717,8 @@ confirmation before any snapshots are deleted. To suppress confirmation, use the
 |-------------------------------|-------------|
 | -h, --help                    | Show this help message and exit. |
 | `--include-inactive`          | Include completed snapshots. |
-| `--database-url DATABASE_URL` | Specify the database URL for the CLI to use. This should only be used as an override to make the CLI talk to a specific instance and isn't expected to be needed. If you are on the Spark plan and want the CLI to use the default instance use the `--use-default-rtdb` flag instead. If neither of the `--database-id` or `--use-default-rtdb` flags are used with the `init` command, the CLI uses the default url. |
+| `--debuggee-id DEBUGGEE_ID`   | Specify the debuggee ID. It must be an ID obtained from the list_debuggees command. This value is required, it must be specified either via this command line argument or via the `SNAPSHOT_DEBUGGER_DEBUGGEE_ID` environment variable.  When both are specified, the value from the command line takes precedence. |
+| `--database-url DATABASE_URL` | Specify the database URL for the CLI to use. This should only be used as an override to make the CLI talk to a specific instance and isn't expected to be needed. If you are on the Spark plan and want the CLI to use the default instance use the --use-default-rtdb flag instead. If neither of the --database-id or --use-default-rtdb flags are used with the init command, the CLI uses the default url.  This value may be specified either via this command line argument or via the `SNAPSHOT_DEBUGGER_DATABASE_URL` environment variable.  When both are specified, the value from the command line takes precedence. |
```

Comment:
Oh wow, now I REALLY want this .md content to be autogenerated from the script itself.

Reply:
Yeah, it would be nice to add, but also there's a few other things related to this.

1. The README is already quite long, I'm wondering if we should simply remove the command reference section as it's the same information as presented when using --help with the CLI. Also, once we add the logpoint support, that will effectively double the number of supported commands.
2. Assuming we keep it, after some initial churn I expect this part would be fairly stable.